src/CarbonAware.WebApi/src/README.md

```diff
@@ -168,22 +168,27 @@ The forecast data represents what the data source predicts future marginal carbo
 This endpoint is useful for determining if there is a more carbon-optimal time to use electicity predicted in the future.
```

Comment:
Add a note that the endpoint will always retrieve the entire current forecast but can filter out forecast data points as requested.